app/models/snack.rb

```diff
@@ -0,0 +1,8 @@
+class Snack < ApplicationRecord
+  has_many :machine_snacks
+  has_many :machines, through: :machine_snacks
+
+  def cost
+    '%.2f' % price
+  end
```

Comment:
would have left price alone but needed the second decimal place for viewer so defined it as 'cost'